system/loggerd/encoderd.cc

```diff
@@ -50,6 +51,11 @@ void encoder_thread(EncoderdState *s, const LogCameraInfo &cam_info) {
   std::vector<std::unique_ptr<Encoder>> encoders;
   VisionIpcClient vipc_client = VisionIpcClient("camerad", cam_info.stream_type, false);
 
+  std::unique_ptr<JpegEncoder> jpeg_encoder;
+  if (cam_info.stream_type == VISION_STREAM_ROAD) {
```

Comment:
let's use EncoderInfo.thumbnail name and make sure to support any encoder having a thumbnail defined